rest-api/core-rest/src/test/java/uk/gov/gchq/gaffer/rest/service/v2/OperationServiceV2IT.java

```diff
@@ -60,8 +100,71 @@ public void shouldReturnOperationDetailFieldsWithClass() throws IOException {
         assertTrue(response.readEntity(String.class).contains(expectedFields));
     }
 
+    @Test
+    public void shouldReturnAllOperationsAsOperationDetails() throws IOException {
```

Comment:
This is going to difficult to maintain. Can you avoid hardcoding all the operation classes here?